applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c

```diff
@@ -75,7 +75,7 @@ static bool initialized;
 static bool delete_broadcast_src;
 
 #if (CONFIG_AURACAST)
-NET_BUF_SIMPLE_DEFINE(pba_buf, BT_UUID_SIZE_16 + 2);
+NET_BUF_SIMPLE_DEFINE(pba_buf, BT_UUID_SIZE_16 + 8);
```

Comment:
Nitpick: I'm wondering if we can have a comment shows how we get this "8", since even the following meta are well commented, but user might forget to enlarge the NET_BUF when trying to add more data.